pkg/api/util/names.go

```diff
@@ -24,6 +24,7 @@ import (
 	"regexp"
 )
 
+// ComputeName is ...
```

Comment:
I'm open to suggestions for this family of short things that do something, but what I don't really know